app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -3,24 +3,40 @@
 miqHttpInject(angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap', 'ManageIQ']))
 .controller('containerTopologyController', ContainerTopologyCtrl);
 
-ContainerTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService'];
+ContainerTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService', '$window'];
+
+function ContainerTopologyCtrl($scope, $http, $interval, $location, topologyService, $window) {
+  $(function($) {
+    $(document).ready(function() {
+      miqHideSearchClearButton();
+    });
+  });
+
+  // Override breadcrumb height. Space is expensive in this page.
+  breadcrumb = document.getElementsByClassName("breadcrumb")[0]
+  if (breadcrumb != null) breadcrumb.style.marginBottom = '0px';
```

Comment:
What are you really trying to accomplish here? Feels like nothing should really change the breadcrumb margin at all...